clients/drcachesim/analyzer_multi.cpp

```diff
@@ -422,7 +422,8 @@ analyzer_multi_tmpl_t<RecordType, ReaderType>::analyzer_multi_tmpl_t()
                 nullptr, op_verbose.get_value(), op_jobs.get_value(),
```

Comment:
> Relaxes raw2trace error reporting to allow some recoverable errors during libipt decode. 

Why "some"?  Which ones?  Wouldn't we want to continue past *all* non-fatal errors if we could and they're really non-fatal?  If not, which ones are we selecting as non-fatal?

Reply:
That's what raw2trace was ending up doing. The new PR description tries to make the distinction between the following two clearer:
- 1-instr discontinuities that we try to continue past and still generate a syscall trace
- full syscall conversion failure where we give up on that syscall trace